pom.xml

```diff
@@ -20,10 +20,12 @@
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-rest</artifactId>
+			<version>3.1.5</version>
```

Comment:
You can omit specifying version in the spring boot dependencies if you have parent